travis/apis.py

```diff
@@ -51,6 +51,8 @@ def __init__(self):
         self._token = os.environ.get("WEBLATE_TOKEN")
         self.host = os.environ.get(
             "WEBLATE_HOST", "https://weblate.vauxoo.com/api")
+        self.ssh = os.environ.get(
+            "WEBLATE_SSH", "ssh://weblate@weblate.vauxoo.com/app/data/vcs")
```

Comment:
@moylop260 
What do you think if we replace it with this `ssh://user@webpage.com` ?
In order to show an example of the format